modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -315,7 +320,7 @@ For example, to use the second option, with an on-premises server you could depl
 mule -M-Dsecret.key=toSecretPassword -M-Denv=prod -M-Ddb.password=secretPassword -app myApp.zip
 ----
 
-In this case, all the values typed into the command are stored only in memory--they are never stored in any file.
+In this case, all the values typed into the command are stored only in memory, they are never stored in any file.
```

Comment:
Semi-colons separate grammatically complete sentences.  ...in memory; these values .....
(because it's not 100% clear what "they" refers to